examples/c/ex_backup_block.c

```diff
@@ -123,14 +127,16 @@ setup_directories(void)
          * For incremental backups we need 0-N. The 0 incremental directory will compare with the
          * original at the end.
          */
-        (void)snprintf(buf, sizeof(buf), "rm -rf %s.%d && mkdir %s.%d", home_incr, i, home_incr, i);
+        (void)snprintf(buf, sizeof(buf), "rm -rf %s.%d && mkdir %s.%d && mkdir %s.%d/%s", home_incr,
+          i, home_incr, i, home_incr, i, logpath);
```

Comment:
This could be shortened using one `mkdir -p` instead of two `mkdir` commands.

Reply:
Fixed them both.